### ISSUE TITLE
Could it be used with git diff?

### ISSUE THREAD

**Issue by vorou** [jingweno/ccat#31]
I ran that:

```
git config --global core.pager ccat
```

And I've got the highlight in diff, but with some weird symbols:

![i](http://i.imgur.com/WwDyaoQ.png)

I guess it the git diff colors, since it looks nice if I turn it off:

![i](http://i.imgur.com/nXKbDZT.png)

I wonder could I combine gitdiff colors (basically, red/green for -/+ lines) and ccat ones?

**Comment by jingweno** [comment]
Your color may be on for diff that's why `ccat` outputs the color code from `git-diff`. Try turning it off with: `git config --global color.diff off`.

**Comment by vorou** [comment]
Yep, that's exactly what I did to get the second screenshot :panda_face: 

It would be cool to combine the two somehow. Like GitHub does:
![i](http://i.imgur.com/HvgQ67t.png)

**Comment by jingweno** [comment]
> It would be cool to combine the two somehow. Like GitHub does

That will prob. include changes to https://github.com/sourcegraph/syntaxhighlight to properly parse diffs.
